CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## newVersion
 * **BUGFIX** (by @imaNNeoFighT): Fix tooltip issue on negative bar charts, #978.
 * **IMPROVEMENT** (by @imaNNeoFighT): Use Container to draw axis-based charts border.
+* **FEATURE** Add the ability to select the RadarChart shape (circle or polygon)
```

Comment:
We started to mention the contributor names. Please mention your name here (like what we did in the above changelogs)